tdrs-backend/tdpservice/users/test/test_api/test_set_profile.py

```diff
@@ -17,21 +17,28 @@ def test_set_profile_data(api_client, user):
     """Test profile data can be set."""
     api_client.login(username=user.username, password="test_password")
     response = api_client.patch(
-        "/v1/users/set_profile/",
+        "/v1/users/user/set_profile/",
         {"first_name": "Joe", "last_name": "Bloggs"},
         format="json",
     )
     assert response.status_code == status.HTTP_200_OK
     assert response.data == {
         "id": user.id,
-        "email": user.username,
         "first_name": "Joe",
         "last_name": "Bloggs",
+        "email": user.username,
         "stt": None,
         "region": None,
+        "login_gov_uuid": user.login_gov_uuid,
+        "hhs_id": user.hhs_id,
         "roles": [],
+        "groups": [],
+        "is_superuser": False,
+        "is_staff": False,
+        "last_login": response.data["last_login"],
```

Comment:
Is there a better way to get this without using the response?

Reply:
probably not. you could `del response.data['last_login']` but i think what you have is fine.